.github/checkgroup.yml

```diff
@@ -39,76 +35,91 @@ subprojects:
       - "pl-cpu (macOS-11, lightning, 3.10, 1.12)"
       - "pl-cpu (ubuntu-20.04, lightning, 3.10, 1.12)"
       - "pl-cpu (windows-2022, lightning, 3.10, 1.12)"
-      - "pytorch-lightning (GPUs)"
-      - "pytorch-lightning (HPUs)"
-      - "pytorch-lightning (IPUs)"
-      - "pl-cpu (slow, macOS-11, pytorch, 3.7, 1.11)"
-      - "pl-cpu (slow, ubuntu-20.04, pytorch, 3.7, 1.11)"
-      - "pl-cpu (slow, windows-2022, pytorch, 3.7, 1.11)"
-      # TODO: since this job has intermittent availability, it cannot be required or it will block all PL PRs from forks
-      #- "test-on-tpus"
-
-  - id: "pytorch_lightning: CPU workflow"
-    paths:
-      - ".github/workflows/ci-pytorch-tests.yml"
-    checks:
-      - "pl-cpu (macOS-11, pytorch, 3.8, 1.10)"
-      - "pl-cpu (macOS-11, pytorch, 3.9, 1.11)"
-      - "pl-cpu (macOS-11, pytorch, 3.10, 1.12)"
-      - "pl-cpu (macOS-11, pytorch, 3.10, 1.13, pre)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.8, 1.10)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.9, 1.11)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.10, 1.11)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.10, 1.12)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.9, 1.13, pre)"
-      - "pl-cpu (windows-2022, pytorch, 3.9, 1.11)"
-      - "pl-cpu (windows-2022, pytorch, 3.10, 1.11)"
-      - "pl-cpu (windows-2022, pytorch, 3.10, 1.12)"
-      - "pl-cpu (windows-2022, pytorch, 3.8, 1.13, pre)"
-      - "pl-cpu (macOS-11, lightning, 3.10, 1.12)"
-      - "pl-cpu (ubuntu-20.04, lightning, 3.10, 1.12)"
-      - "pl-cpu (windows-2022, lightning, 3.10, 1.12)"
-
-  - id: "pytorch_lightning: Slow workflow"
-    paths:
-      - ".github/workflows/ci-pytorch-tests-slow.yml"
-    checks:
       - "pl-cpu (slow, macOS-11, pytorch, 3.7, 1.11)"
       - "pl-cpu (slow, ubuntu-20.04, pytorch, 3.7, 1.11)"
       - "pl-cpu (slow, windows-2022, pytorch, 3.7, 1.11)"
 
   - id: "pytorch_lightning: Azure GPU"
     paths:
       - ".azure/gpu-tests-pytorch.yml"
-      - "tests/tests_pytorch/run_standalone_*.sh"
+      # only the azure GPU workflow runs the examples
+      # all examples don't need to be added because they aren't used in CI, but these are
+      - "examples/run_pl_examples.sh"
+      - "examples/pl_basics/backbone_image_classifier.py"
+      - "examples/pl_basics/autoencoder.py"
+      - "examples/pl_fault_tolerant/automatic.py"
+      - "examples/test_pl_examples.py"
+      - "examples/pl_integrations/dali_image_classifier.py"
+      - "requirements/pytorch/**"
+      - "src/pytorch_lightning/**"
+      - "tests/tests_pytorch/**"
+      - "setup.cfg"  # includes pytest config
+      - "requirements/lite/**"
+      - "src/lightning_lite/**"
+      - ".actions/**"
```

Comment:
```suggestion
      - "requirements/pytorch/*"
      - "src/pytorch_lightning/**"
      - "tests/tests_pytorch/**"
      - "setup.cfg"  # includes pytest config
      - "requirements/lite/*"
      - "src/lightning_lite/**"
      - ".actions/*"
```

Reply:
You suggested to add those in https://github.com/Lightning-AI/lightning/pull/15633#discussion_r1020001116! I'll make the change again but choose one :joy: